ios/FluentUI/Card Nudge/CardNudge.swift

```diff
@@ -225,7 +225,7 @@ class MSFCardNudgeStateImpl: NSObject, ObservableObject, Identifiable, MSFCardNu
 }
 
 #if DEBUG
-struct CardNudge_Previews: PreviewProvider {
+struct CardNudgePreviews: PreviewProvider {
```

Comment:
Oops yeah, I have to make this change locally because SwiftLint gives me an error otherwise: 
Type Name Violation: Type name should only contain alphanumeric characters: 'CardNudge_Previews' (type_name)

I usually uncheck the file in GitHub so it doesn't get included in my change, but must've missed it this time. Is this error a mistake or should it actually be renamed?